lib/github-auth.js

```diff
@@ -205,9 +206,11 @@ function rmGithubToken(token) {
 // Personal tokens allow access to GitHub private repositories.
 // You can manage your personal GitHub token at
 // <https://github.com/settings/tokens>.
-if (serverSecrets && serverSecrets.gh_token) {
-  addGithubToken(serverSecrets.gh_token);
-}
+tokensPromise.then(function(tokens) {
+  if (serverSecrets && serverSecrets.gh_token) {
+    addGithubToken(serverSecrets.gh_token);
+  }
+});
```

Comment:
All `then(…)` should have a `.catch(…)`.

But here, I believe there is no need to have a `then(…)` at all. What is its purpose?